edb/lib/ext/graphql.edgeql

```diff
@@ -17,6 +17,8 @@
 #
 
 
+CREATE EXTENSION PACKAGE graphql VERSION '1.0';
+
 CREATE MODULE stdgraphql;
```

Comment:
Should this line be nested in `CREATE EXTENSION PACKAGE`?

Reply:
I haven't actually implemented nested DDL for extensions, as we need to add a new tracking mechanism to attribute created objects to extensions for it to make sense, and that, in turn, requires a bit of refactoring.  Can be done post beta.